tools/tpm2_createprimary.c

```diff
@@ -178,16 +179,16 @@ int create_primary(TSS2_SYS_CONTEXT *sapi_context) {
     sessionsData.cmdAuthsCount = 1;
     sessionsDataOut.rspAuthsCount = 1;
 
-    inSensitive.t.sensitive.data.t.size = 0;
-    inSensitive.t.size = inSensitive.t.sensitive.userAuth.b.size + 2;
+    ctx.inSensitive.t.sensitive.data.t.size = 0;
```

Comment:
instead of using TPM2B_EMPTY_INIT, can you create a new macro for this that sets both size fields to 0?
```
TPM2B_SENSITIVE_CREATE_EMPTY_INIT
```